passes/opt/opt_expr.cc

```diff
@@ -953,6 +953,10 @@ void replace_const_cells(RTLIL::Design *design, RTLIL::Module *module, bool cons
 			}
 
 			if (b.is_fully_const()) {
+				if (b.is_fully_undef()) {
```

Comment:
Should this be `is_fully_undef` and not `is_any_undef`? My reading of the standard is that *any* x or z bits in any operand cause the result to be x.
![Screenshot_20190913_114627](https://user-images.githubusercontent.com/54771/64860249-1fa75880-d61c-11e9-92b1-7be4825851e2.png)

Reply:
This code is only for 1 bit expressions, so fully_undef and any_undef is the same.. However, answering the more general point, for example `2'b00 == 2'b1x` is `1'b0`. That's what's meant by "if [...] the relation is ambiguous".